packages/warehouses/src/warehouseClients/SnowflakeWarehouseClient.ts

```diff
@@ -210,6 +214,10 @@ export class SnowflakeWarehouseClient extends WarehouseBaseClient<CreateSnowflak
                 token: credentials.token,
                 authenticator: 'OAUTH',
             };
+        } else if (credentials.authenticationType === 'external_browser') {
```

Comment:
Why not use the constant for comparison?

Reply:
yeah, makes sense. I'll update this and all other authenticationtypes